packages/rocketchat-accounts/.npm/package/README

```diff
@@ -0,0 +1,7 @@
+This directory and the files immediately inside it are automatically generated
```

Comment:
I have no clue why I got those files gor generated, but as suggested, I committed them along with the actual changes. They should do no harm